File: exercicios/Python/1150/1150.py
x = int(input())
z = x - 1
i = 2
S = 1
soma = x
while z <= x:
    z = int(input())
while soma <= z:
    soma = soma + x + S
    
    if soma <= z:
        i = i + 1
        S = S + 1
print(i)

File: exercicios/R/1150/1150.r
input = file('stdin','r')

i = as.integer(2)
s = as.integer(1)

x = as.integer(readLines(input,n=1))

z = as.integer(x - 1)

soma = as.integer(x)

while(z <= x){
    z = as.integer(readLines(input,n=1))
}
while(soma <= z){
    soma = soma + x + s

    if(soma <= z){
        i = i +1
        s = s + 1
    }
}

write(i,'')